components/table/src/simple-columns/favorite-column.html

```diff
@@ -0,0 +1,30 @@
+<ng-container
+  [dtColumnDef]="name"
+  dtColumnAlign="icon"
+  [dtColumnProportion]="proportion"
+>
+  <ng-container *dtHeaderCellDef>
+    <dt-header-cell [dt-sort-header]="sortable" start="desc"
+      >{{ label }}</dt-header-cell
+    >
+  </ng-container>
+  <dt-cell
+    *dtCellDef="let data"
+    [dtIndicator]="_getIndicator(data)"
+    [dtIndicatorColor]="_getIndicator(data)"
+    [attr.title]="_getData(data)"
+  >
+    <ng-container
+      *ngTemplateOutlet="favoriteTemplate; context:{ favorite: _getData(data), data: data }"
+    ></ng-container>
+  </dt-cell>
+</ng-container>
+
+<ng-template #favoriteTemplate let-favorite="favorite" let-data="data">
+  <dt-icon
+    [name]="favorite ? 'favorite' : 'star-empty'"
+    [class.dt-favorite-column-filled-star]="favorite"
+    [class.dt-favorite-column-empty-star]="!favorite"
+    (click)="onChange(data)"
```

Comment:
clicking on non button/anchor elements i kinda bad for a11y. Can we change it to an icon button?

Reply:
I changed to a button but please check the scss to know if it's a better way to do the coloring of the star